components/backup/src/endpoint.rs

```diff
@@ -611,7 +611,7 @@ impl SoftLimitKeeper {
 /// The endpoint of backup.
 ///
 /// It coordinates backup tasks and dispatches them to different workers.
-pub struct Endpoint<E: Engine, R: RegionInfoProvider + Clone + 'static> {
+pub struct Endpoint<E: Storage, R: RegionInfoProvider + Clone + 'static> {
```

Comment:
```suggestion
pub struct Endpoint<S: Storage, R: RegionInfoProvider + Clone + 'static> {
```
replace all `E: Storage` with `S: Storage`